inference_lib/src/aqlm/inference_kernels/cuda_kernel.cpp

```diff
@@ -0,0 +1,61 @@
+#include <torch/all.h>
```

Comment:
Strong opinion: we need compilation instructions OR a promise that you'll add them with a fixed deadline.

If i missed the instructions somewhere, please direct me to them